packages/sanity/src/core/i18n/bundles/studio.ts

```diff
@@ -68,6 +68,310 @@ export const studioLocaleStrings = {
   'timeAgo.seconds.minimal': '{{count}}m',
   /* Relative time, granularity: seconds, using a minimal format, configured to show ago suffix*/
   'timeAgo.seconds.minimal.ago': '{{count}}m ago',
+
+  /** --- Nav bar --- */
+
+  /** Label for workspace picker */
+  'navbar.action.select-workspace': 'Select workspace',
+
+  /** Label to open search action when the search is on full screen */
+  'navbar.action.open-search': 'Open search',
+
+  /** --- Worksace menu --- */
+
+  /** Title for Workplaces dropdown menu */
+  'navbar.workspace-menu.title': 'Workspaces',
+
+  /** Title for "community" link within the workspace menu */
+  'navbar.workspace-menu.community-title': 'Community',
+
+  /** Title for "docs" link within the workspace menu */
+  'navbar.workspace-menu.docs-title': 'Docs',
+
+  /** Title for "privacy" link within the workspace menu */
+  'navbar.workspace-menu.privacy-title': 'Privacy',
+
+  /** Title for "sanity io website" link within the workspace menu */
+  'navbar.workspace-menu.sanity-io-title': 'sanity.io',
+
+  /** Label for area section that indicates that you can choose your workspace */
+  'navbar.workspace-menu.choose-your-workspace-label': 'Choose your workspace',
+
+  /** Label for action to choose a different workspace */
+  'navbar.workspace-menu.action.choose-another-workspace': 'Choose another workspace',
+
+  /** Label for action to add a workspace */
+  'navbar.workspace-menu.action.add-workspace': 'Add workspace',
+
+  /** --- New Document --- */
+
+  /** Placeholder for the "search" input within the new document menu */
+  'navbar.new-document.search': 'Search',
```

Comment:
**question:** should we consider creating a common path / namespace for single words? Something like `Search` is bound to be used across various points in the studio. This applies to other words too, such as `Loading`, `Clear`,  `True`, `False`, etc

Reply:
This is something I have brought up with @rexxars in the past but we weren't able to get to a specific conclusion before the hackaton and leave.

Because these are used cross packages (and that means across different languages and potentially different use cases) it's a hard one.

Here's why: In portuguese there might be some cases where the word that is most commonly translated to "Search" is used only in some cases but not in others where _technically_ in english it would all end up being the same word. There are some other languages which I'm sure have even more details of these kinds which I'm concerned we're being blind to. At the same time, adding more keys can add more noise (and more business expense to those that are looking to translate the studio).